lib/utils/logger-utility.js

```diff
@@ -27,26 +27,32 @@ module.exports = (function () {
       for (let item of logBuffer) {
         // display separators for each item before-hand except the first time
         if (item !== logBuffer[0]) {
+          console.warn();
           console.warn("----------------------------------------");
         }
         item = JSON.parse(item);
-
-        console.warn("[" + item.time + "] - " + LEVEL_MAPPING[item.level] + " - " + item.activity.toUpperCase());
-        if (item["request-id"]) {
-          console.warn("request-id: " + item["request-id"]);
-        }
-        console.warn(item.request.method + " " + item.request.url);
-        console.warn("status code: " + item.response.statusCode + " " + item.response.statusMessage);
+        const requestId = (item["request-id"])? item["request-id"] : "";
+        const requestMethodUrl = (item.request)? item.request.method + " " + item.request.url : "";
+        const statusCode = (item.response)? item.response.statusCode + " " + item.response.statusMessage : "";
+        const requestHeaders = (item.request)? JSON.stringify(item.request.headers) : "{}";
+        const requestbody = (item.request && item.request.body)? item.request.body : "{}";
+        const responseHeaders = (item.response)? JSON.stringify(item.response.headers): "{}";
+        const responseBody = (item.body)? JSON.stringify(item.body): "{}";
+        console.warn("[%s] - %s - %s", item.time, LEVEL_MAPPING[item.level], item.activity.toUpperCase());
+        console.warn("%s", requestMethodUrl);
+        console.warn("request-id: %s", requestId);
```

Comment:
nit: fine as is but better with a quote to distinguish "" from "    ":
```
console.warn("request-id: '%s'", requestId);
```

Reply:
since it was not printing the quotes before, let's push this in for now.